src/main/java/it/pagopa/pn/deliverypush/action/CompletionWorkFlowHandler.java

```diff
@@ -108,7 +113,7 @@ private void sendSimpleRegisteredLetter(NotificationInt notification, Integer re
             log.info("Sending simple registered letter  - iun {} id {}", notification.getIun(), recIndex);
             externalChannelService.sendNotificationForRegisteredLetter(notification, physicalAddress, recIndex);
         } else {
-            log.info("Simple registered letter can't be send, there isn't physical address for recipient. iun {} id {}", notification.getIun(), recIndex);
+            log.error("Simple registered letter can't be send, there isn't physical address for recipient. iun {} id {}", notification.getIun(), recIndex);
```

Comment:
é diventato veramente un errore?

Reply:
Si, perchè con la logica attuale il passaggio di stato in delivered avviene solo se viene inviata la simple registered letter. Dal momento che il physical address è obbligatorio, se per qualche motivo arriva qui senza tale indirizzo è un errore va segnalato e analizzato

Reply:
ok